crypto/asn1/x_long.c

```diff
@@ -12,6 +12,9 @@
 #include <openssl/asn1t.h>
 #include <openssl/bn.h>
 
+#if !(OPENSSL_API_COMPAT < 0x10200000L)
```

Comment:
Is the idea that OPENSSL_API_COMPAT will always be used with the '<' operator?  (Should that get written down somewhere?)